delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -1,5 +1,7 @@
 package ch.epfl.bluebrain.nexus.delta.plugins.storage.files
 
+import _root_.retry.CatsEffect._
```

Comment:
do we need this _root_ ?

Reply:
Intellij seems to change this when organizing imports

Reply:
It never did it for me. But let's remove it, since it isn't necessary